oak/server/oak_runtime.cc

```diff
@@ -75,12 +78,26 @@ asylo::Status OakRuntime::Initialize(const ApplicationConfiguration& config) {
                            "Node at end of channel not found");
     }
 
-    LOG(INFO) << "Create channel " << src_name << "." << src_port << " -> " << dest_name << "."
-              << dest_port;
-    auto channel = std::make_shared<MessageChannel>();
-
-    src_node->AddNamedChannel(src_port, absl::make_unique<MessageChannelWriteHalf>(channel));
-    dest_node->AddNamedChannel(dest_port, absl::make_unique<MessageChannelReadHalf>(channel));
+    if (dest_node == log_node && logging_channel != nullptr) {
+      // Special case for all configured channels going to the logging
```

Comment:
What should we do about this long term? Should we have multiple producer / single consumer channels? or allow duplicating handles in some other way (e.g. https://fuchsia.dev/fuchsia-src/zircon/syscalls/handle_duplicate.md )?